app/assets/javascripts/statistics.js

```diff
@@ -64,6 +71,18 @@ var getScale = function getScale() {
   return scale
 }
 
+var prohibitStep = function prohibitStep(elementSelector) {
```

Comment:
@shir it's not working correctly for my case. One state can be false or true, and I need add class hide to one element and delete from another, but the state in this toggleClass work another way, he adds the only class if true, and remove only if false.